sample.env

```diff
@@ -0,0 +1,2 @@
+WALLETCONNECT_PROJECT_ID=f58c0242def84c3b9befe9b1e6086bbd
+WALLET_CONNECT_RELAY_URL=wss://relay.walletconnect.com
```

Comment:
unused?

Reply:
Removed in [056ea41](https://github.com/shapeshift/relay-claiming-cli/pull/1/commits/056ea41c5ba3e13a6e20689f0f2bf9eb5ce2af5d)